### DISCUSSION TITLE
Error 401 with tweets plugin

### DISCUSSION THREAD

**Discussion by JayantGoel001** [lowlighter/metrics#142] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/54479676/104272631-9203a380-54c3-11eb-9a32-2548b37dee77.png)

Hey @lowlighter , I  am getting this error.

**Comment by lowlighter** [comment, marked as answer]
Hello ! 

You're getting this error since the tweets plugin died and you enabled `plugins_errors_fatal`.

Below is the logs of the plugin:
```
metrics/compute/JayantGoel001/plugins > tweets > completed (error)
{
  name: 'tweets',
  result: {
    error: {
      message: 'API returned 401',
      instance: {
        title: 'Unauthorized',
        type: 'about:blank',
        status: 401,
        detail: 'Unauthorized'
      }
    }
  }
}
```

From what I see, it's a 401 (Unauthorized) HTTP code, which usually means the provided Twitter token is not valid.
Seems your `${{ secrets.TWITTER_TOKEN }}` is at least non empty, but I suggest that you double-check it in case you added whitespace or forgot a character

**Comment by JayantGoel001** [comment]
@lowlighter  Ya It was error related to twitter API only now its working Great.
Thank a Lot
